python/lsst/pipe/analysis/plotUtils.py

```diff
@@ -416,6 +419,45 @@ def bboxToRaDec(bbox, wcs):
     return ra, dec
 
 
+def pointToRaDec(point, wcs, returnUnits="degrees"):
```

Comment:
Heavy-weight for what it does.  You can do this as one line of code anyway:
`ra, decl = tractWcs.pixelToSky(tractBbox.getMinX(), tractBbox.getMinY()).getPosition(units=afwGeom.degrees)`
if you do `tract00 = tractWcs.pixelToSky...` you can access the RA of `tract00` like `tract00[0]` or `tract00.getX()`